lms/djangoapps/certificates/api.py

```diff
@@ -91,6 +91,42 @@ def get_certificates_for_user(username):
     ]
 
 
+def get_certificate_for_user(username, course_key):
```

Comment:
Can you update `get_certificates_for_user` above to use this code, or is that not feasible?